.eslintrc.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
+    "globals": {
+        "document": true,
+        "window": true,
+        "MutationObserver": true,
+        "requestIdleCallback": true,
+        "waitIdle": true,
+        "wait": true,
+        "fetch": true,
+        "fetchCoverage": true,
+        "fetchAnnotate": true,
+        "fetchChangesetCoverage": true,
+        "isCoverageSupported": true,
+        "injectToggle": true,
+        "getPath": true,
+        "gitToHg": true,
+        "getNavigationPanel": true,
+        "SUPPORTED_EXTENSIONS": true
+    },
+    "env": {
+        "es6": true,
+        "node": true,
+        "webextensions": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 8,
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "sourceType": "module"
+    },
+    "rules": {
```

Comment:
Some of these rules might already be defined by `eslint-plugin-mozilla`, we should make sure we don't overwrite them.

Reply:
To use those rules, I think we should add `extends: 'plugin:mozilla/recommended'` in the .eslintrc.

Reply:
Here are the rules defined by default by the mozilla plugin: https://dxr.mozilla.org/mozilla-central/source/tools/lint/eslint/eslint-plugin-mozilla/lib/configs/recommended.js